app/api/api_v1/endpoints/assignment_router.py

```diff
@@ -26,11 +29,21 @@ class UpdateAssignmentBody(BaseModel):
     available_date: datetime | None
     due_date: datetime | None
     is_published: bool = UNSET
+    manual_grading: bool = UNSET
 
-class GradingBody(BaseModel):
+class OtterGradingBody(BaseModel):
     master_notebook_content: str
     otter_config_content: str
 
+class ManualGrade(BaseModel):
+    submission_id: int
+    # Between [0,1]
+    grade_proportion: float
```

Comment:
What is this proportion? Does it mean how much the grade is weighted in the class towards the final grade, or?

Reply:
It's the student's grade.

Reply:
It was renamed from grade percent after harit misinterpreted grade percent to be in [0,100].

Reply:
Oohh this is manual grading, that's right, my bad. Okay this makes sense.